Tests/AppTests/BuildTriggerTests.swift

```diff
@@ -956,8 +957,9 @@ class BuildTriggerTests: AppTestCase {
         let deleteCount = try await trimBuilds(on: app.db)
 
         // validate
-        XCTAssertEqual(deleteCount, 5)
-        try await XCTAssertEqualAsync(try await Build.query(on: app.db).all().map(\.id), [.id0, .id1, .id4])
+        XCTAssertEqual(deleteCount, 3)
+        try await XCTAssertEqualAsync(try await Build.query(on: app.db).all().map(\.id),
+                                      [.id0, .id1, .id2, .id3, .id4])
```

Comment:
This shows that the new trimBuilds _can't_ be responsible for the increase in build candidates. It's deleting _fewer_ builds than before, so if anything it would lead to _lowering_ the number of build candidates.